packages/material-ui-lab/src/SpeedDial/SpeedDial.d.ts

```diff
@@ -7,18 +7,61 @@ import { TransitionHandlerProps } from '@material-ui/core/transitions';
 export interface SpeedDialProps
   extends StandardProps<
     React.HTMLAttributes<HTMLDivElement> & Partial<TransitionHandlerProps>,
-    SpeedDialClassKey
+    SpeedDialClassKey,
+    'children'
   > {
+  /**
+   * SpeedDialActions to display when the SpeedDial is `open`.
+   */
+  children?: React.ReactNode;
+  /**
+   * The aria-label of the `Button` element.
+   * Also used to provide the `id` for the `SpeedDial` element and its children.
+   */
   ariaLabel: string;
+  /**
+   * Props applied to the [`Button`](/api/button/) element.
+   */
   ButtonProps?: Partial<ButtonProps>;
+  /**
+   * The direction the actions open relative to the floating action button.
+   */
   direction?: 'up' | 'down' | 'left' | 'right';
+  /**
+   * If `true`, the SpeedDial will be hidden.
+   */
   hidden?: boolean;
-  icon: React.ReactNode;
-  onClose?: React.ReactEventHandler<{}>;
+  /**
+   * The icon to display in the SpeedDial Floating Action Button. The `SpeedDialIcon` component
+   * provides a default Icon with animation.
+   */
+  icon?: React.ReactNode;
```

Comment:
Conflict: PropTypes uses Element which means it's required, the type definitions uses ReactNode which makes it optional

Reply:
Looks good like this